website/server/models/challenge.js

```diff
@@ -209,8 +209,22 @@ schema.methods.updateTask = async function challengeUpdateTask (task) {
     updateCmd.$set[key] = syncableAttrs[key];
   }
 
```

Comment:
Yeah that's a good point, it shouldn't ever be undefined and if it is, it's better to fail to save so we can investigate instead of just not saving the particular fields. I would agree that it's probably better to fail fast than to fail silently